samples/xrc/myframe.cpp

```diff
@@ -285,6 +286,14 @@ void MyFrame::OnUncenteredToolOrMenuCommand(wxCommandEvent& WXUNUSED(event))
     dlg.ShowModal();
 }
 
+void MyFrame::OnMultipleAccels(wxCommandEvent& WXUNUSED(event))
+{
+    wxString msg;
+    msg.Printf( "You can open this dialog also with 'Ctrl-Q' and 'Shift-Ctrl-W' accelerators.");
```

Comment:
But actually it has `Ctrl-W` as main accelerator too, so I'd rather say
```suggestion
    msg.Printf( "You can open this dialog with any of `Ctrl-W` (main), 'Ctrl-Q' or 'Shift-Ctrl-W' (extra) accelerators.");
```